osu.Game/Skinning/SkinnableSound.cs

```diff
@@ -37,6 +37,8 @@ public class SkinnableSound : SkinReloadableDrawable, IAdjustableAudioComponent
         /// </remarks>
         protected bool PlayWhenZeroVolume => Looping;
 
+        protected virtual bool PlayWhenPaused => false;
```

Comment:
I'm not sure I entirely agree with how this is done, given that `UnpausableSkinnableSound` has to exist. Thus we have a base class that allows sample playback control via an `ISamplePlaybackDisabler`, but then a derived class that opts out of that behaviour. The name is also pretty... oxymoronic?

Ideally I'd invert this and have the base `SkinnableSound` be the unpausable one, and the derived to DI in the disabler and do the necessary pause/unpause logic as the more specific class. Maybe that's a code quality PR for later, though... Might as well rename to something like `CanBePaused`/`Disabled` or something.

Reply:
Hmm, that does indeed sound better. Wasn't 100% sure about this. I guess the only argument I have for doing it this way is there's only one usage of the derived case so I felt comfortable making it the more derived class..

@smoogipoo thoughts?

Reply:
That sounds like a good path forward.